test/puppetlabs/puppetdb/integration/db_resilience/db_fallback.clj

```diff
@@ -0,0 +1,22 @@
+(ns puppetlabs.puppetdb.integration.db-resilience.db-fallback
```

Comment:
Just verifying here @mullr, I didn't see much value in the postgres_restart test, is that why you didn't port it over?